src/app/components/candidate-request/candidate-request.component.html

```diff
@@ -58,24 +66,34 @@ <h2>{{ 'request.header' | translate }}</h2>
       <mat-form-field appearance="outline">
         <mat-label>{{ 'request.language' | translate }}</mat-label>
         <mat-select formControlName="sandboxPreferredLanguageId">
-          <mat-option *ngFor="let lang of sandboxChosen.languages" [value]="lang !== null ? lang.id : ''">{{lang !== null ?
-            lang.name : ''}}</mat-option>
+          <mat-option
+            *ngFor="let lang of sandboxChosen.languages"
```

Comment:
it's easier to filter list and remove nulls than add ? : expressions. I would like to see Pipe for this purpose. Can you create one? It's not complicated (spoiler)